src/modules/navigator/mission.cpp

```diff
@@ -447,11 +568,13 @@ Mission::set_mission_items()
 		if (_mission_type != MISSION_TYPE_NONE) {
 
 			if (_navigator->get_land_detected()->landed) {
-				mavlink_log_info(_navigator->get_mavlink_log_pub(), "Mission finished, landed.");
+				mavlink_log_info(_navigator->get_mavlink_log_pub(),
```

Comment:
Can we log the mission execution direction more explicitly?

Reply:
I added the mission execution type to the mission_result message so it is logged now.